src/components/ItemDetails/ItemDetails.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import * as S from './Style'
+import { DETAILS_TYPE } from '../../common/styledType'
+
+const ItemDetails = ({ data, type }) => {
+    const { 
+        item, 
+        details 
+    } = DETAILS_TYPE[type]
+
+    const {
+        _id,
+        cost,
+        date,
+        category,
+        __v
```

Comment:
fo not use it, it is mongodb internal property, not for use on client I suppose

Reply:
Could you make private this in request?